doc/releases/changelog-dev.md

```diff
@@ -51,6 +51,27 @@
   Also added ``qml.pauli.trace_inner_product`` that can handle batches of dense matrices.
   [(#6811)](https://github.com/PennyLaneAI/pennylane/pull/6811)
 
+* Added class ``qml.FromBloq`` that takes Qualtran bloqs and translates them into equivalent PennyLane operators.
+  For example, we can now import Bloqs and use them in a way similar to how we use PennyLane templates:
+
+  ```python
+  from qualtran.bloqs.basic_gates import ZPowGate
```

Comment:
Please make sure to run the examples locally and assert that it works as expected.

Reply:
Yes it works locally. Maybe there can be a better example. @DSGuala What do you think?